packages/design-system-scripts/jest/a11y.global-setup.js

```diff
@@ -1,27 +1,15 @@
 require('regenerator-runtime/runtime');
 const StaticServer = require('static-server');
 const chalk = require('chalk');
-const childProcess = require('child_process');
 const log = require('fancy-log');
 
 const APP_PORT = 3001;
 
-function buildApp() {
-  log(chalk.green('\nBuilding docs site in production mode...\n'));
-  // Build files in production while ignoring rootPath
-  childProcess.execSync(
-    process.env.BUILD_COMMAND || 'yarn build-docs --skipLatest --ignoreRootPath',
-    {
-      stdio: ['ignore', 'ignore', process.stderr],
-    }
-  );
-  log(chalk.green('done ✓'));
-}
-
 async function startServer() {
-  log(chalk.green('\nStarting local server hosting production build...'));
+  console.log('\n');
+  log(chalk.green('Starting local web server in ./storybook-static/ ...'));
   const server = new StaticServer({
-    rootPath: process.env.BUILD_PATH,
+    rootPath: './storybook-static/',
```

Comment:
I like this setup 👍. Now that I know that the config file gets loaded before this one executes, this makes a lot of sense.